release/ray_release/tests/test_wheels.py

```diff
@@ -51,20 +51,20 @@ def test_get_ray_wheels_url(remove_buildkite_env):
         repo_url="https://github.com/ray-project/ray.git",
         branch="master",
         commit="1234",
-        ray_version="3.0.0.dev0",
+        ray_version="2.8.0",
```

Comment:
all occurrences under `release/ray_release/tests` are for unit testing, please revert.